guessingGame.js

```diff
@@ -17,12 +21,25 @@ for (let letter of wordToGuess) {
   console.log(`Current word is: ${game.currentWord}`);
   console.log();
 }
+*/
+
+while (guessingGameIsDone(game) === false){
+  let guess = readlineSync.question(' Guess a letter: ');
+  guessingGameMakeGuess(game,guess);
+  console.log(`\n Current word is ${game.currentWord}`);
```

Comment:
If you want an extra blank line, I'd do a `console.log()` on a separate line. A newline character at the start of the string is a little weird.